### DISCUSSION TITLE
Наклейки

### DISCUSSION THREAD

**Discussion by Trelexx** [metrostroi-repo/MetrostroiAddon#587] [General]
Можете пожайлуста сказать как добавить наклейки на сервер, если можно то дайте файл самих наклеек

**Comment by SmileDoge** [comment]
какие самые наклейки?

**Reply by Trelexx** [reply]
Не прислоняться и уступать пожилым и т.д.

**Reply by Trelexx** [reply]
ответ будет?

**Reply by Trelexx** [reply]
@SmileDoge

**Reply by SmileDoge** [reply]
делаешь модель, текстуру, прописывааешь в поезде

**Reply by Trelexx** [reply]
можно узнать где сами наклейки, они же вроде есть в этом аддоне
